README.rst

```diff
@@ -17,7 +17,7 @@ pfcon  v1.3.3
 Overview
 ********
 
-This repository provides ``pfcon`` -- a controlling service that speaks to remote ``pman`` and ``pfioh`` services.
+This repository provides ``pfcon`` -- a controlling service that speaks to remote ``pman`` and ``pfioh`` services. This is the bleeding edge branch.
```

Comment:
What is this extra line about?  Did that only apply to the dev branch?

Reply:
Yes -- pretty much. I'll remove